WooCommerce/Classes/Authentication/Store Creation/Plan/StoreCreationPlanView.swift

```diff
@@ -0,0 +1,214 @@
+                            Text(Localization.planTitle)
+                                .fontWeight(.semibold)
+                                .font(.title3)
+                                .foregroundColor(.white)
+
+                            // Price information.
+                            HStack(alignment: .bottom) {
+                                Text(viewModel.plan.displayPrice)
+                                    .fontWeight(.bold)
+                                    .foregroundColor(.white)
+                                    .largeTitleStyle()
+                                Text(Localization.priceDuration)
+                                    .foregroundColor(Color(.secondaryLabel))
+                                    .bodyStyle()
+                            }
+                        }
+                        .padding(.horizontal, insets: .init(top: 0, leading: 24, bottom: 0, trailing: 0))
+
+                        Spacer()
+
+                        Image(uiImage: .storeCreationPlanImage)
+                    }
+
+                    Divider()
+                        .frame(height: Layout.dividerHeight)
+                        .foregroundColor(Color(Layout.dividerColor))
+                        .padding(.horizontal, insets: Layout.defaultPadding)
+
+                    VStack(alignment: .leading, spacing: 0) {
+                        Spacer()
+                            .frame(height: 8)
+
+                        // Header label.
+                        Text(Localization.subtitle)
+                            .fontWeight(.bold)
+                            .foregroundColor(Color(.white))
+                            .titleStyle()
+
+                        Spacer()
+                            .frame(height: 16)
+
+                        // Powered by WPCOM.
+                        HStack(spacing: 5) {
+                            Text(Localization.poweredByWPCOMPrompt)
+                                .foregroundColor(Color(.secondaryLabel))
+                                .footnoteStyle()
+                            Image(uiImage: .wpcomLogoImage)
+                        }
+
+                        Spacer()
+                            .frame(height: 32)
+
+                        // Plan features.
+                        VStack(alignment: .leading, spacing: 16) {
+                            ForEach(viewModel.features, id: \.title) { feature in
+                                HStack(spacing: 12) {
+                                    Image(uiImage: feature.icon)
+                                        .renderingMode(.template)
+                                        .foregroundColor(Color(.wooCommercePurple(.shade90)))
+                                    Text(feature.title)
+                                        .foregroundColor(Color(.label))
+                                        .bodyStyle()
+                                }
+                            }
+                        }
```

Comment:
Nit: What do you think about breaking the `body` down into smaller views? For example, this part can be a separate `featuresListView` View.

Reply:
sure 👍 updated in https://github.com/woocommerce/woocommerce-ios/pull/8120/commits/2ddae25ecd7de029d7115a71b884e784f4fb7ed2